src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -366,9 +376,11 @@ def _build_workflow_engine_evidence_data(
             "detector_id": self.detector.id,
             "value": evaluation_value,
             "data_packet_source_id": str(data_packet.source_id),
-            "conditions": [
-                result.condition.get_snapshot() for result in evaluation_result.condition_results
-            ],
+            "conditions": (
+                [result.condition.get_snapshot() for result in evaluation_result.condition_results]
+                if evaluation_result is not None
+                else []
```

Comment:
hmm, should we be invoking this method if we don't have an evaluation result? this is used to create the evidence to say _why_ a detector was triggered, if we don't have an evaluation result, then the detector is not triggered and we should be able to early exit evaluation for that group.